spark/src/test/scala/io/substrait/spark/TPCHPlan.scala

```diff
@@ -178,6 +176,39 @@ class TPCHPlan extends TPCHBase with SubstraitPlanTestBase {
         "((l_orderkey, L_COMMITDATE), (l_orderkey, L_COMMITDATE, l_linestatus), l_shipdate, ())")
   }
 
+  test("in_subquery") {
+    assertSqlSubstraitRelRoundTrip(
+      "select p_retailprice from part where p_comment in (select o_comment from orders)")
+  }
+
+  test("exists") {
```

Comment:
just note for any future reference: at least `in_subquery` above and this pass already w/o this PR. Though this `exists` test produces wrong nullability for the scalar subquery in proto roundtrip..

Reply:
Yes, it did already pass, but I wanted to add an explicit test for it, since I was fiddling in this area.